docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,93 @@
+
+## Feed Settings
+
+### Information
+
+These should be auto-filled when adding a feed, but can be modified after. **Visibility** will define if the feed is displayed in the main feed, only in category feeds, or not at all.
+
+### Archival
+
+This section will let you override the default settings for feed archiving and update frequency.
+
+### Login
+
+Some feeds require a username/password submitted over HTTP. These usually aren't needed for feeds.
+
+### Advanced
+
+#### Retrieve a truncated feed from within FreshRSS
+
+This question comes up regularly, so we'll try to clarify how one can retrieve a truncated RSS feed with FreshRSS. Please note that the process is absolutely not user friendly, but it works. :)
+
+Please be aware that this way you'll generate much more traffic to the originating sites, and they might block you accordingly. FreshRSS performance is also negatively affected, because you'll have to fetch the full article content one by one. So it's a feature to use sparingly!
+
+What's meant by "CSS path of articles on the original site" actually corresponds to the "path" consisting of IDs and classes (which in HTML, matches the id and class attributes) to retrieve only the interesting part that corresponds to the article. Ideally, this path starts with an id (which is unique to the page).
+
+##### Example: Rue89
+
+To find this path, you have to go to the address of one of the truncated articles (for example, http://www.rue89.com/2013/10/15/prof-maths-jai-atteint-lextase-dihn-pedagogie-inversee-246635). You look have to look for the "block" of HTML that corresponds to article content (in the source code!).
+
+Here we find that the block that encompasses nothing but the content of the article is ```<div class="content clearfix">```. We'll only use the `.content` class here. Nevertheless, as said above, it's best to start the path with an id. If we go back to the parent block, we find ```<div id="article">``` and that's perfect! The path will be ```#article .content```.
+
+##### Add the corresponding classes to the article CSS path on the feed configuration page. Examples:
+
+*  Rue89: ```#article .content```
+*  PCINpact: ```#actu_content```
+*  Lesnumériques: ```article#body div.text.clearfix```
+*  Phoronix: ```#main .content```
+
+##### Combining CSS Classes
+Let's say we have an article which contains ads within its content, and we do not want to have those ads retrieved by FreshRSS. Example Source Code:
+```
+<div id='article'>
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<p class='ad'>unwanted content</p>
+<h2>wanted</h2>
+<p class='content'>wanted content</p>
+<h2>wanted</h2>
+<p class='ad'>unwanted content</p>
+<p class='content'>wanted content</p>
+</div>
+```
+In this case it's possible to combine multiple CSS selectors with a comma: ```#article p.content, #article h2```
+
+#### Retrieve a truncated feed with external tools
+
+Complimentary tools can be used to retrieve full article content, such as:
```

Comment:
Complimentary means that RSS-Bridge keeps saying what a nice application FreshRSS is. :-P
```suggestion
Complementary tools can be used to retrieve full article content, such as:
```